mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -113,16 +113,23 @@ def __remote_reset(self):
         self.logger.prn_inf("remote resources reset...")
         if not self.selected_resource:
             raise Exception("remote resource not exists!")
-        if not self.selected_resource.reset():
-            raise Exception("remote resources reset failed!")
+        try:
+            self.selected_resource.reset()
```

Comment:
this actually breaks backward compatibility with previous remote module version. Do we need it anymore?

Reply:
also this extra try-catch doesn’t help at all since catch is done already by caller:
https://github.com/ARMmbed/htrun/blob/e1da9332c54454bc64bc508a948ec6ed3142f46c/mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py#L84
it only hides original reason, which is not good..